dotnet/samples/KernelSyntaxExamples/Example73_Pipelining.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel;
+
+#pragma warning disable RCS1110 // Declare type inside namespace.
+
+public static class Example73_Pipelining
+{
+    /// <summary>
+    /// Provides an example of combining multiple functions into a single function that invokes
+    /// them in a sequence, passing the output from one as input to the next.
+    /// </summary>
+    public static async Task RunAsync()
+    {
+        KernelFunction parseInt32 = KernelFunctionFactory.CreateFromMethod((string s) => double.Parse(s, CultureInfo.InvariantCulture), "parseInt32");
```

Comment:
Theorically this should work also for Prompt Functions, does it?

Reply:
No, sounds like a bug. Will take a look over the weekend.